src/main/clojure/clara/rules/engine/impl.cljc

```diff
@@ -0,0 +1,45 @@
+(ns clara.rules.engine.impl)
```

Comment:
Ok,

I use this convention mainly because most of the time there are more than just protocols.
Could not guessed that we would end up there :)

It's a global rename not a big deal to do. Will do it Wednesday.

Sent from my iPhone

> On Jun 19, 2015, at 11:25, Ryan Brush <notifications@github.com> wrote:
> 
> In src/main/clojure/clara/rules/engine/impl.cljc:
> 
> > @@ -0,0 +1,45 @@
> > +(ns clara.rules.engine.impl)
> It looks like this is all protocol definitions. A convention I've seen elsewhere is to simply name this namespace "clara.rules.engine.protocols", making it clear that's what it is. Putting protocols into their own namespace is a good pattern since it makes interactive reloading easier, so I like that these are separated out.
> 
> —
> Reply to this email directly or view it on GitHub.
>